addons/ai/XEH_preInit.sqf

```diff
@@ -6,4 +6,16 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+// Fix rotating garrisoned units
+["ModuleCurator_F", "Init", {
+    params ["_logic"];
+
+    _logic addEventHandler ["CuratorObjectEdited", {
+        params ["", "_object"];
+        if (_object getVariable [QGVAR(garrisoned), false]) then {
+            _object doWatch ((ASLtoAGL eyePos _object) vectorAdd (vectorDir _object));
```

Comment:
`doWatch` is argument local.

Reply:
Ah yeah, this version of the garrison module actually works for non-local units :sweat_smile: